inputs.css

```diff
@@ -0,0 +1,15 @@
+.text-input {
+    background-color: var(--gray-1);
+    border-radius: 2em;
+    padding-top: 0.8em;
+    padding-bottom: 0.8em;
+    font-size: 0.9em;
+    border: 0;
+    padding-left: 2.5em;
+}
+
+.text-input-with-icon svg {
```

Comment:
I think we should also have a rule that does:

```
.text-input-with-icon {
  position: relative;
}
```

This way, all the code is encapsulated here. And we can remove the `relative` class from the HTML, in case of these elements.